substrate/frame/src/lib.rs

```diff
@@ -326,9 +339,11 @@ pub mod testing_prelude {
 	/// Other helper macros from `frame_support` that help with asserting in tests.
 	pub use frame_support::{
 		assert_err, assert_err_ignore_postinfo, assert_error_encoded_size, assert_noop, assert_ok,
-		assert_storage_noop, ensure, hypothetically, storage_alias,
+		assert_storage_noop, hypothetically, storage_alias, DefaultNoBound,
```

Comment:
why is `ensure` removed? `DefaultNoBound` should already be in prelude through derive IIRC